README.md

```diff
@@ -45,27 +52,72 @@ Valid `type` values are `tar` and `zip` while available URL replacement tokens a
 - `{type}`: the package type, e.g. `library`
 - `{checksum}`: the SHA1 checksum for the distribution archive of this version
 
+## Development setup
+
 ### Using Docker Compose
 
-The setup procedure can be simplified by using Docker Compose.
+Alternatively, you can build a development site using [Docker](https://www.docker.com/get-docker) and 
+[Docker Compose](https://docs.docker.com/compose/) with the provided configuration.
 
-Requirements:
+Docker provides the necessary services and tools such as a web server and a database server to get the site running, 
```

Comment:
It doesn't provide a database server.

Reply:
A drupal site is created with this process. Are you sure these comments are valid ?